src/containers/Installer.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+
+import InstallerView from '../components/InstallerView'
+import { installApp } from '../actions'
+
+const mapStateToProps = (state, ownProps) => ({
+  slug: state.slug,
+  appUrl: state.appUrl
+})
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  onSubmit: (slug, appUrl, isUpdate) => {
+    return dispatch(installApp(slug, appUrl, isUpdate))
```

Comment:
In ES2017 arrow function, we can avoid `return` statement if we have only one instruction.

> Note that an arrow function with a block body does not automatically return a value. Use a return statement for that.
— https://hacks.mozilla.org/2015/06/es6-in-depth-arrow-functions/

```js
onSubmit: (slug, appURL, isUpdate) => dispatch(installApp(slug, appURL, isUpdate))
```